.circleci/docker/common/install_rocm.sh

```diff
@@ -14,7 +18,12 @@ install_magma() {
     cp make.inc-examples/make.inc.hip-gcc-mkl make.inc
     echo 'LIBDIR += -L$(MKLROOT)/lib' >> make.inc
     echo 'LIB += -Wl,--enable-new-dtags -Wl,--rpath,/opt/rocm/lib -Wl,--rpath,$(MKLROOT)/lib -Wl,--rpath,/opt/rocm/magma/lib' >> make.inc
-    echo 'DEVCCFLAGS += --amdgpu-target=gfx900 --amdgpu-target=gfx906 --amdgpu-target=gfx908 --amdgpu-target=gfx90a --amdgpu-target=gfx1030 --gpu-max-threads-per-block=256' >> make.inc
+    if [[ $(ver $ROCM_VERSION) -lt $(ver 4.3) ]]; then
```

Comment:
This is why I closed the PR.  Our QA team doesn't set ROCM_VERSION when building the docker image.  They use `sed` to change various install paths, rather than set ROCM_VERSION.  It's stuck at 3.5.1, because it works for them in their current workflow.

Reply:
@jithunnair-amd

Reply:
Could we use the same source for ROCm version we use in LoadHIP.cmake?https://github.com/ROCmSoftwarePlatform/pytorch/blob/master/cmake/public/LoadHIP.cmake#L154